firewood/src/merkle/node.rs

```diff
@@ -45,6 +46,19 @@ bitflags! {
     }
 }
 
+const NIBBLES_PER_BYTE: usize = 2;
+const BITS_PER_NIBBLE: u64 = 4;
+const BITS_PER_BYTE: u64 = 8;
```

Comment:
Just an FYI, you can do math at compile time.
Also, why are the types different here? Might make sense to add a comment.